docs/TESTING-DEPLOYER.md

```diff
@@ -251,14 +248,14 @@ retag() { # Usage: retag FROM_IMAGE_WITH_TAG TO_IMAGE_WITH_TAG
   docker pull $FROM && docker tag $FROM $TO && docker push $TO
 }
 retagall() { # Usage: retagall FROM_REGISTRY FROM_TAG TO_REGISTRY TO_TAG
-  local FROM=$1 TO=$2 FROM_TAG=$3 TO_TAG=$4; local -; set -eu
-  retag $FROM:$FROM_TAG                                         $TO:$TO_TAG
-  retag $FROM/cert-manager-acmesolver:$FROM_TAG                 $TO/cert-manager-acmesolver:$TO_TAG
-  retag $FROM/cert-manager-cainjector:$FROM_TAG                 $TO/cert-manager-cainjector:$TO_TAG
-  retag $FROM/cert-manager-webhook:$FROM_TAG                    $TO/cert-manager-webhook:$TO_TAG
-  retag $FROM/cert-manager-google-cas-issuer:$FROM_TAG          $TO/cert-manager-google-cas-issuer:$TO_TAG
-  retag $FROM/preflight:$FROM_TAG                               $TO/preflight:$TO_TAG
-  retag gcr.io/cloud-marketplace-tools/metering/ubbagent:latest $TO/ubbagent:$TO_TAG
+  local FROM=$1 TO=$2 FROM_TAG=$3 TO_TAG=$4
+  retag $FROM:$FROM_TAG $TO:$TO_TAG || exit 1
+  retag $FROM/cert-manager-acmesolver:$FROM_TAG $TO/cert-manager-acmesolver:$TO_TAG || exit 1
+  retag $FROM/cert-manager-cainjector:$FROM_TAG $TO/cert-manager-cainjector:$TO_TAG || exit 1
+  retag $FROM/cert-manager-webhook:$FROM_TAG $TO/cert-manager-webhook:$TO_TAG || exit 1
+  retag $FROM/cert-manager-google-cas-issuer:$FROM_TAG $TO/cert-manager-google-cas-issuer:$TO_TAG || exit 1
+  retag $FROM/preflight:$FROM_TAG $TO/preflight:$TO_TAG || exit 1
+  retag gcr.io/cloud-marketplace-tools/metering/ubbagent:latest $TO/ubbagent:$TO_TAG || exit 1
```

Comment:
Seems unrelated to the rest of this PR, but perhaps `set -o errexit` would be clearer at the top of the function.

Reply:
I used `set -o errexit` initially, but then I realized that it affected your current shell, and since I tell people to copy-paste this command into their shell, I don't want them to get their shell suddenly crash due to my `set -o errexit` 😅

I found out that Bash 4.4 has [local -](https://stackoverflow.com/questions/46480686/how-to-restrict-shell-options-to-a-function-using-local) that makes these options "local" to the current scope. But it only works if the user who copy-pastes the above command is using Bash (won't work with Zsh, Dash...)